go.mod

```diff
@@ -203,14 +203,6 @@ replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 // allows us to specify that as an option.
 replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
 
-// Temporary replace rule until kvdb is tagged with changes from
-// https://github.com/lightningnetwork/lnd/pull/8611.
-replace github.com/lightningnetwork/lnd/kvdb => ./kvdb
```

Comment:
I think the `kvdb` replace needs to remain until we also pushed a new tag of that?

Reply:
+1

Reply:
Re-added!